larpandoracontent/LArHelpers/LArHitWidthHelper.h

```diff
@@ -281,10 +281,31 @@ class LArHitWidthHelper
      *  @param  constituentHitVector the input vector of contituent hits
      *  @param  lowerXCoordinate the lower x extremal point
      *  @param  higherXCoordinate the higher x extremal point
-     *
      */
     static void GetExtremalCoordinatesX(const ConstituentHitVector &constituentHitVector, pandora::CartesianVector &lowerXCoordinate,
         pandora::CartesianVector &higherXCoordinate);
+
+    /**
+     *  @brief  Consider the hit width to find the closest position of a calo hit to a specified line
+     *
+     *  @param  lineStart the start position of the line 
+     *  @param  lineDirection the direction of the line
+     *  @param  pCaloHit the input calo hit
+     *
+     *  @return  CartesianVector the closest position
+     */
+    static pandora::CartesianVector GetClosestPointToLine2D(const pandora::CartesianVector &lineStart, const pandora::CartesianVector &lineDirection,
+        const pandora::CaloHit *const pCaloHit);
+
+    /**
+     *  @brief  Consider the hit width to find the smallest of a calo hit to a given point
```

Comment:
Is "distance" missing in the brief? I.e. "find the smallest distance of a calo hit..."